scss/mixins.scss

```diff
@@ -290,11 +285,24 @@ $_o-brand-define-readme: "https://github.com/Financial-Times/o-brand/#obranddefi
     @for $character-index from 1 through str-length($name) {
         $character: str-slice($name, $character-index, $character-index);
         @if $character == " " {
-            @error '#{$error-context}. #{$thing} "#{$name}" is invalid. #{$thing} names can not contain spaces.';
+            @return _oBrandError('#{$error-context}. #{$thing} "#{$name}" is invalid. #{$thing} names can not contain spaces.');
         }
         @if not index($valid-characters, $character) {
-            @error '#{$error-context}. #{$thing} "#{$name}" is invalid. #{$thing} names can only contain lowercase alphanumeric characters and "-".';
+            @return _oBrandError('#{$error-context}. #{$thing} "#{$name}" is invalid. #{$thing} names can only contain lowercase alphanumeric characters and "-".');
         }
     }
     @return true;
 }
+
+/// Error handler that allows for errors to be tested in dev environments
+///
+/// @access private
+/// @param {string} $message Error message
+/// @param {boolean} $override whether or not to output an error or a string to compare against
+@function _oBrandError($message, $override: $_test-environment) {
```

Comment:
`$_test-environment` should use the o-brand prefix, but I'm not sure I understand the value of allowing compilation to continue after errors in a test environment?